Extensions.Caching.PostgreSql/DatabaseOperations.cs

```diff
@@ -43,6 +44,7 @@ public DatabaseOperations(IOptions<PostgreSqlCacheOptions> options, ILogger<Data
             SqlCommands = new SqlCommands(cacheOptions.SchemaName, cacheOptions.TableName);
 
             this._logger = logger;
+            this._updateOnGetCacheItem = !cacheOptions.DisableUpdateOnGetCacheItem;
```

Comment:
When changing the name to `UpdateOnGetCacheItem`  and the default value to `true` there is no need to negate anymore, 🚀